### PR TITLE
Type definition

### PR DESCRIPTION
**Pull request by cedroux**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,9 +41,9 @@ We are willing to answer questions.
 ## Getting Started
 
 ```javascript
-import vjsn from "zome-vjsn";
+import vueSpatialNavigation from "zome-vjsn";
 
-app.use(vjsn, globalConfig);
+app.use(vueSpatialNavigation);
```
Comment (BlueBazze) [inline README.md]: globalConfig is useful and shouldnt be left out.

File: README.md [hunk 2]
```diff
@@ -41,9 +41,9 @@ We are willing to answer questions.
 ## Getting Started
 
 ```javascript
-import vjsn from "zome-vjsn";
+import vueSpatialNavigation from "zome-vjsn";
```
Comment (BlueBazze) [inline README.md]: Why the change?

File: lib/index.ts [hunk 3]
```diff
@@ -20,7 +21,7 @@ const vueSpatialNavigation = {
     // Vue.prototype.$SpatialNavigation = SpatialNavigation;
     Vue.config.globalProperties.$SpatialNavigation = SpatialNavigation;
 
-    const assignConfig = (sectionId, config) => {
+    const assignConfig = (sectionId: string | undefined, config: object) => {
```
Comment (BlueBazze) [inline lib/index.ts]: config is optional. And should be the type of `vjsnOptions` if im not mistaken

**Review by BlueBazze — CHANGES_REQUESTED** [review]
Sorry, havent seen your changes before now.
There are just a few questions i have commented.
Thank you for your interest, let me know if we can improve the package

File: README.md (hunk 1 above)
Comment (cedroux) [reply README.md]: `globalConfig` was not declared in the previouses lines (undefined error), so I removed it, as it was optional.

File: lib/index.ts (hunk 3 above)
Comment (cedroux) [reply lib/index.ts]: You're right, fixed it! 🔥

File: README.md (hunk 2 above)
Comment (cedroux) [reply README.md]: I replaced `import vjsn` with `import vueSpatialNavigation` because its the currently exported name ([see here](https://github.com/Syncronet-APS/zome-vjsn/blob/80ff7c71dcd37e07d94ef7788006ceb1661dd8a1/lib/index.ts#L153))

Comment (BlueBazze) [reply README.md]: I dont have any protest against this.
My original intention was to split from spacerefugee's original package.
vueSpatialNavigation was annoying to write out so it was shortened.

File: lib/options.ts [hunk 4]
```diff
@@ -97,7 +97,12 @@ It should be a PlainObject consists of four properties: `'left'`, `'right'`, `'u
 **Note:** Assigning an empty string to any of these properties makes SpatialNavigation go nowhere at that direction.
 
    */
-  leaveFor?: any;
+  leaveFor?: {
+    left?: string;
+    right?: string;
+    up?: string;
+    down?: string;
+  };
```
Comment (BlueBazze) [inline lib/options.ts:108]: It seems that it can also be an array of or a single DOM element.
I yeeted JQuery, not sure if i got all of it out.
https://github.com/luke-chang/js-spatial-navigation#leavefor

**Review by BlueBazze — CHANGES_REQUESTED** [review]
Looks good

File: lib/options.ts (hunk 4 above)
Comment (BlueBazze) [reply lib/options.ts:108]: Take a look at this commit. Also added a license comment on the definition file you made.
https://github.com/Syncronet-APS/zome-vjsn/commit/14e8157854ecf8f644ea1d122be50d349d36c684

File: README.md (hunk 2 above)
Comment (cedroux) [reply README.md]: Oh alright I understood, my IDE was tricking me into thinking there was an error with just `vjsn` because this name wasn't defined anywhere. But since it is the default export we can name it whatever we want. I will revert the changes, sorry!

File: lib/options.ts (hunk 4 above)
Comment (cedroux) [reply lib/options.ts:108]: Shouldn't the `LICENCE` file be enough?

> If it is not possible or desirable to put the notice in a particular file, then You may include the notice in a location (such as a LICENSE file in a relevant directory) where a recipient would be likely to look for such a notice.

https://www.mozilla.org/en-US/MPL/2.0/
